docs/sources/developers/plugins/sign-a-plugin.md

```diff
@@ -5,6 +5,93 @@ type = "docs"
 
 # Sign a plugin
 
-Signing a plugin allows Grafana to verify the authenticity of the plugin with [signature verification]({{< relref "../../plugins/plugin-signature-verification.md" >}}). This gives users a way to make sure plugins haven't been tampered with. All Grafana Labs-authored backend plugins, including Enterprise plugins, are signed.
+Signing a plugin allows Grafana to verify the authenticity of the plugin with [signature verification]({{< relref "../../plugins/plugin-signatures.md" >}}). This gives users a way to make sure plugins haven't been tampered with. All Grafana Labs-authored backend plugins, including Enterprise plugins, are signed.
 
-We're looking into providing a process for allowing community plugins to be signed in an upcoming version of Grafana.
+Plugin signature verification (signing) is a security measure to make sure plugins haven't been tampered with. Upon loading, Grafana checks to see if a plugin is signed or unsigned when inspecting and verifying its digital signature.
+
+> **Important:** Future versions of Grafana will require all plugins to be signed.
+
+## Sign your plugin using Grafana Toolkit
+
+The easiest way to sign your plugin is by using the [Grafana Toolkit](https://www.npmjs.com/package/@grafana/toolkit).
+
+You can sign your plugin as a _public_ or a _private_ plugin. In both cases, you need to [create an account on Grafana.com](https://grafana.com/signup) and generate an API key with the `PluginPublisher` role. By creating an account, you can verify that you own the plugin that you want to sign.
+
+### Sign a public plugin
+
+Plugins signed under the community or commercial signature level are considered _public plugins_. Public plugins are published on [Grafana Plugin](https://grafana.com/plugins). For more information about installing public plugins, refer to [Install Grafana plugins]({{< relref "../../plugins/installation.md" >}}).
+
+1. Request a plugin signature level by sending an email to [XXX@grafana.com](mailto:XXX@grafana.com).
```

Comment:
plugins@grafana.com goes to the helpdesk today, we can use that or create a new address just for signature requests.